Apps/WebClient/src/ClientApp/src/views/profile.vue

```diff
@@ -527,6 +539,9 @@ export default class ProfileView extends Vue {
         var oidcUserPromise = authenticationService.getOidcUserProfile();
         var userEmailPromise = this.getUserEmail({ hdid: this.user.hdid });
         var userSMSPromise = this.getUserSMS({ hdid: this.user.hdid });
+        var userPreferencePromise = this.getUserPreference({
+            hdid: this.user.hdid,
+        });
         var userProfilePromise = this.userProfileService.getProfile(
```

Comment:
since the profile contains the user information that we have, lets move the preferences to the same object. That way a single call will load both

Reply:
I'm refactoring this now.